aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIO.java

```diff
@@ -719,6 +723,84 @@ public void multipleStorageCredentialsConfigured() {
         .hasMessage("Invalid S3 Credentials: only one S3 credential should exist");
   }
 
+  @Test
+  public void initFileIOWithoutCredentials() {
+    S3FileIO fileIO = new S3FileIO();
+    fileIO.initialize(ImmutableMap.of("client.region", "us-east-1"));
+
+    S3ServiceClientConfiguration actualConfiguration = fileIO.client().serviceClientConfiguration();
+    assertThat(actualConfiguration).isNotNull();
+    try {
+      actualConfiguration.credentialsProvider().resolveIdentity();
+    } catch (SdkClientException e) {
```

Comment:
please use `assertThatThrownBy(..).isInstanceOf(..).hasMessage(..)` instead of a try-catch block

Reply:
> please use `assertThatThrownBy(..).isInstanceOf(..).hasMessage(..)` instead of a try-catch block

thanks for your advice, updated.